tools/napi/symbol_exports.json

```diff
@@ -0,0 +1,143 @@
+{
+  "symbols": [
+    "node_api_create_syntax_error",
```

Comment:
Symbols to be exported are now defined in this JSON file. The `#[napi_sym]` macro checks for missing entries and panics. `./tools/napi/generate_link_win.js` is used to generate the LINK `cli/exports.def` on Windows, which is also checked into git.

To add a new napi function:
1. Place `#[napi_sym]` on top of your implementation.
2. Add the function's identifier to this JSON list.
3. Finally, run `./tools/napi/generate_link_win.js` to update `cli/exports.def`.

Reply:
Can you add this as a module docstring?